charts/redpanda/values.yaml

```diff
@@ -550,7 +550,22 @@ config:
     # tm_sync_timeout_ms: 2000ms                                   # Time to wait state catch up before rejecting a request
     # tm_violation_recovery_policy: crash                          # Describes how to recover from an invariant violation happened on the transaction coordinator level
     # transactional_id_expiration_ms: 10080min                     # Producer ids are expired once this time has elapsed after the last write with the given producer ID
-  tunable: {}
+  tunable:
+    log_segment_size: 134217728                                    # 128 mb
+    log_segment_size_min: 16777216                                 # 16 mb
+    log_segment_size_max: 268435456                                # 256 mb
+    kafka_batch_max_bytes: 1048576                                 # 1 mb
+    topic_partitions_per_shard: 1000
+    compacted_log_segment_size: 67108864                           # 64 mb
+    max_compacted_log_segment_size: 536870912                      # 512 mb
+    kafka_connections_max: 15100
+    kafka_connection_rate_limit: 1000
+    partition_autobalancing_mode: "continuous"
```

Comment:
continuous autobalancing requires a license, so should not be on by default unless helm can know whether it's installing with a license.

Reply:
You can just check that one of these is set: https://github.com/redpanda-data/helm-charts/blob/e228c875180d5aa6ac3918de45de02cc8b2583d3/charts/redpanda/values.yaml#L45-L46